src/main/surfrdUtilsMod.F90

```diff
@@ -183,78 +183,78 @@ subroutine collapse_crop_types(wt_cft, fert_cft, cftsize, begg, endg, verbose, s
 
     if (cftsize > 0) then  ! The opposite applies only if use_fates
 
```

Comment:
All changes in this file are from indenting the lines within the outer if statement.

Reply:
Good